schemes/musica/tuvx/musica_ccpp_tuvx.F90

```diff
@@ -199,6 +231,9 @@ subroutine tuvx_run(temperature, dry_air_density,                 &
       if (errcode /= 0) return
     end do
 
+    call set_surface_albedo_values( surface_albedo_profile, surface_albedo, errmsg, errcode )
```

Comment:
maybe good to note that surface_albedo here is w.r.t. shortwave, direct for clarity.

Reply:
Good idea, added.

Reply:
Agreed with @mattldawson - UV is the more accurate term. I was basing off the comments in the old CAM but they could be updated to be consistent with what TUV-x photolysis actually expects. Thanks!

Reply:
Updated, thank you, both of you!